rust_src/src/threads.rs

```diff
@@ -24,6 +25,14 @@ impl ThreadStateRef {
     pub fn name(self) -> LispObject {
         LispObject::from(self.name)
     }
+
+    #[inline]
+    pub fn alive_p(self) -> LispObject {
```

Comment:
Methods in the Rust code need to follow Rust naming standards instead of elisp standards. This method should be named `is_alive`.

While the match code below is valid, an if/else would match the rest of the code better.

Consider having this routine return a bool and performing the t/nil conversion in the `lisp_fn`. Then you do not need an if/else here.